### PR TITLE
draft CC Alerts doc

### LINKED ISSUE #7461: Document CockroachCloud alerting system
Document the new alerts signup feature in CC.

- List of metrics to alert on: https://docs.google.com/spreadsheets/d/1DoKM2F3fMtGNygBk_8AAdu9xTs4d0tL2kXpdRFIXvSg/edit#gid=0
- Wireframes: https://www.figma.com/file/iA5L17xXeM4farQevK4EUZ/obsrv_admin-ui-alerts?node-id=105%3A209
- Design prototype: https://www.figma.com/proto/iA5L17xXeM4farQevK4EUZ/obsrv_admin-ui-alerts?node-id=225%3A8760&viewport=784%2C670%2C0.1282435804605484&scaling=min-zoom

### LINKED ISSUE #7808: document CockroachCloud alerts
CockroachCloud will offer basic alerts received by email.

Roadmap issue: https://cockroachlabs.atlassian.net/browse/CRDB-293

- Document the alert types
- Document where these are found in the Cloud Console
- Document the flow for toggling and responding to alerts

### PR DESCRIPTION
**Pull request by taroface**
Fixes #7808.
Fixes #7461.

### CODE REVIEW

**Comment by thtruo** [comment]
Nice, these look comprehensive! Have a nit to follow (via reviewable)

**Review by thtruo — COMMENTED** [review]
> Reviewed 1 of 2 files at r1, 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10409)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abarganier and @taroface)

___
*[cockroachcloud/alerts-page.md, line 39 at r2](https://reviewable.io/reviews/cockroachdb/docs/10409#-MYwXMmL5N--pwL5uMCW:-MYwXMmL5N--pwL5uMCX:b-ey7yx9) ([raw file](https://github.com/cockroachdb/docs/blob/9e4d16e63d40a9df0e15cc4255e4035b71caea8a/cockroachcloud/alerts-page.md#L39)):*
> ```Markdown
> - Under **Add Email Recipients**, add at least one email address and click **Add**.
> 
> Note that alerts are enabled for all clusters in your Organization. To filter alerts on specific clusters, you can use an email alias to send alerts to a monitoring tool such as [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) or [Pagerduty](https://www.pagerduty.com/).
> ```

Nit: "Pagerduty" should be spelled "PagerDuty"

**Review by nkodali — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10409)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abarganier and @taroface)

___
*[cockroachcloud/alerts-page.md, line 45 at r2](https://reviewable.io/reviews/cockroachdb/docs/10409#-MYzgWt4D06AhrKlqlGg:-MYzgWt4D06AhrKlqlGh:b-2vp5do) ([raw file](https://github.com/cockroachdb/docs/blob/9e4d16e63d40a9df0e15cc4255e4035b71caea8a/cockroachcloud/alerts-page.md#L45)):*
> ```Markdown
> - Under **Send test email**, select a cluster and alert type.
> - Add at least one email address and click **Add**.
> - Click **Send test email**.
> ```

nit: extra whitespace at end of line

**Review by taroface — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10409)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abarganier)

___
*[cockroachcloud/alerts-page.md, line 45 at r2](https://reviewable.io/reviews/cockroachdb/docs/10409#-MYzgWt4D06AhrKlqlGg:-MYznLIl96pUxgXLq8YO:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/9e4d16e63d40a9df0e15cc4255e4035b71caea8a/cockroachcloud/alerts-page.md#L45)):*
<details><summary><i>Previously, nkodali (Namrata Kodali) wrote…</i></summary><blockquote>

nit: extra whitespace at end of line
</blockquote></details>

Done.

File: cockroachcloud/alerts-page.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**Memory Utilization:**
+
+- Cluster-wide available memory is **20% or less** on *average* for at least 60 minutes.
+- Node-level available memory is **10% or less** on *average* for at least 90 minutes.
+
+If you receive an alert repeatedly, you may need to [optimize your workload](../stable/make-queries-fast.html) or [scale your CockroachCloud cluster](cluster-management.html?filters=dedicated#add-or-remove-nodes-from-a-cluster).
+
+{{site.data.alerts.callout_success}}
+When scaling your cluster, we recommend first scaling machines above 2 vCPUs. If this doesn't sufficiently improve performance, then add more nodes.
```
Comment (mikeCRL) [inline cockroachcloud/alerts-page.md:29]:
```suggestion
When scaling your cluster, we recommend first scaling VMs to include more than 2 vCPUs each. If this doesn't sufficiently improve performance, then add more nodes.
```

I'm hoping this resolves some _slight_ ambiguity and remains accurate.

File: cockroachcloud/alerts-page.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Under **Configure CockroachCloud Alerts**, toggle the **Alerts are on** switch.
+- Under **Add Email Recipients**, add at least one email address and click **Add**.
+
+Note that alerts are enabled for all clusters in your Organization. To filter alerts on specific clusters, you can use an email alias to send alerts to a monitoring tool such as [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) or [PagerDuty](https://www.pagerduty.com/).
+
+To send a test alert:
+
+- Under **Send test email**, select a cluster and alert type.
+- Add at least one email address and click **Add**.
+- Click **Send test email**.
```
Comment (mikeCRL) [inline cockroachcloud/alerts-page.md:45]:
```suggestion
1. Under **Send test email**, select a cluster and an alert type.
1. Add at least one email address and click **Add**.
1. Click **Send test email**.
```

**Review by mikeCRL — APPROVED** [review]
Provisionally approving with a couple suggestions for your review.

**Review by taroface — COMMENTED** [review]
TFTR!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/10409#-:-MZ-4Nbo1sZyK-0oIKv1:b-814g8k)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @abarganier, @mikeCRL, and @thtruo)

___
*[cockroachcloud/alerts-page.md, line 29 at r3](https://reviewable.io/reviews/cockroachdb/docs/10409#-MYvmstW5PbfT6_yD1PS-r3-29:-MZ-4TFY-4xmp7DRN2jz:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/ada18d5cf0087ff379cd7e721b9f73174e85b536/cockroachcloud/alerts-page.md#L29)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

```suggestion
When scaling your cluster, we recommend first scaling VMs to include more than 2 vCPUs each. If this doesn't sufficiently improve performance, then add more nodes.
```

I'm hoping this resolves some _slight_ ambiguity and remains accurate.
</blockquote></details>

Done.
___
*[cockroachcloud/alerts-page.md, line 45 at r3](https://reviewable.io/reviews/cockroachdb/docs/10409#-MYvmstW5PbfT6_yD1PS-r3-45:-MZ-4aQ78mNfLM1AhW6t:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/ada18d5cf0087ff379cd7e721b9f73174e85b536/cockroachcloud/alerts-page.md#L45)):*
<details><summary><i>Previously, mikeCRL (Mike Lewis) wrote…</i></summary><blockquote>

```suggestion
1. Under **Send test email**, select a cluster and an alert type.
1. Add at least one email address and click **Add**.
1. Click **Send test email**.
```
</blockquote></details>

Done.

**Review by thtruo — COMMENTED** [review]
LGTM

**Pull request merged**